tests/services/inventory/test_data/additional_cai_resources.dump

```diff
@@ -12,6 +12,8 @@
 {"name":"//compute.googleapis.com/projects/project2/zones/us-central1-b/autoscalers/gae-default-20161128t212211","asset_type":"compute.googleapis.com/Autoscaler","resource":{"version":"v1","discovery_document_uri":"https://www.googleapis.com/discovery/v1/apis/compute/v1/rest","discovery_name":"Autoscaler","parent":"//cloudresourcemanager.googleapis.com/projects/1042","data":{"autoscalingPolicy":{"coolDownPeriodSec":120,"cpuUtilization":{"utilizationTarget":0.5},"maxNumReplicas":20,"minNumReplicas":2},"creationTimestamp":"2016-11-28T21:27:35.916-08:00","id":"9001","name":"gae-default-20161128t212211","selfLink":"https://www.googleapis.com/compute/v1/projects/project2/zones/us-central1-b/autoscalers/gae-default-20161128t212211","status":"ACTIVE","target":"https://www.googleapis.com/replicapool/v1beta1/projects/project2/zones/us-central1-b/pools/gae-default-20161128t212211","zone":"https://www.googleapis.com/compute/v1/projects/project2/zones/us-central1-b"}}}
 # Compute BackendBucket (id 901x)
 {"name":"//compute.googleapis.com/projects/project2/global/backendBuckets/test-bucket","asset_type":"compute.googleapis.com/BackendBucket","resource":{"version":"v1","discovery_document_uri":"https://www.googleapis.com/discovery/v1/apis/compute/v1/rest","discovery_name":"BackendBucket","parent":"//cloudresourcemanager.googleapis.com/projects/1042","data":{"bucketName":"test-bucket.a.b","creationTimestamp":"2017-10-31T13:18:31.015-07:00","description":"","enableCdn":false,"id":"9011","name":"test-bucket","selfLink":"https://www.googleapis.com/compute/v1/projects/project2/global/backendBuckets/test-bucket"}}}
+# Compute GlobalForwardingRule
+{"asset_type":"compute.googleapis.com/GlobalForwardingRule","name":"//compute.googleapis.com/projects/project1/regions/us-central1/global/forwardingRules/lb-1","resource":{"version": "v1","discovery_document_uri":"https://www.googleapis.com/discovery/v1/apis/compute/v1/rest","discovery_name": "GlobalForwardingRule","parent":"//cloudresourcemanager.googleapis.com/projects/94226340476","data":{"IPAddress":"172.16.1.2","IPProtocol":"TCP","creationTimestamp":"2017-05-05T12:00:01.000-07:00","description":"","id":"1111","kind":"compute#forwardingRule","loadBalancingScheme":"EXTERNAL","name":"lb-1","portRange":"80-80","region":"https://www.googleapis.com/compute/v1/projects/project1/regions/us-central1","selfLink":"https://www.googleapis.com/compute/v1/projects/project1/regions/us-central1/global/forwardingRules/lb-1",      "networkTier": "PREMIUM","target":"https://www.googleapis.com/compute/v1/projects/project1/regions/us-central1/targetHttpProxies/lb-1-target-proxy"},"parent":"//cloudresourcemanager.googleapis.com/projects/1041"}}
```

Comment:
Please change project ID of parent from 94226340476 to 1042 or the crawler will not discover this resource. Remove the second parent entry.

Set the id to 9161 and note in the comment that the ID reserved space is (id 916x). All IDs across all resources under test need to be unique, so we used reserved space for each resource to ensure that is the case.

Reply:
Once this resource is discovered correctly, the crawler test will fail until the resource count is increased for the forwarding rules resource.

Reply:
Thanks so much for the explanation. Updated project ID and id and incremented expected count for forwarding rules resource. But the test fails as the resource is not discovered correctly. Am I missing something?